tests/components/nzbget/test_config_flow.py

```diff
@@ -0,0 +1,156 @@
+"""Test the NZBGet config flow."""
+from pynzbgetapi import NZBGetAPIException
+
+from homeassistant.components.nzbget.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_SCAN_INTERVAL, CONF_VERIFY_SSL
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+from homeassistant.setup import async_setup_component
+
+from . import (
+    ENTRY_CONFIG,
+    USER_INPUT,
+    _patch_async_setup,
+    _patch_async_setup_entry,
+    _patch_history,
+    _patch_status,
+    _patch_version,
+)
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+
+async def test_user_form(hass):
+    """Test we get the user initiated form."""
+    await async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+    assert result["type"] == RESULT_TYPE_FORM
+    assert result["errors"] == {}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)